### PR TITLE
Add 079 Event & Change 939 sound method

### PR DESCRIPTION
**Pull request by warquys**

### CODE REVIEW

File: Synapse3.SynapseModule/Patching/Patches/EventMapPatches.cs [hunk 1]
```diff
@@ -32,6 +33,42 @@ public static bool Scp914Upgrade(Collider[] intake, Vector3 moveVector, Scp914Mo
...
+            return false;
+        }
+        catch (Exception ex)
+        {
+            NeuronLogger.For<Synapse>().Error("Sy3 Event: Trigger Tesla Event failed\n" + ex);
+            return true;
+        }
+    }
+}
+
```
Comment (GrafDimenzio) [inline Synapse3.SynapseModule/Patching/Patches/EventMapPatches.cs]: I already implemented the Trigger Tesla event

Comment (warquys) [reply Synapse3.SynapseModule/Patching/Patches/EventMapPatches.cs]: ok

Comment (warquys) [reply Synapse3.SynapseModule/Patching/Patches/EventMapPatches.cs:71]: but the event add is for 079

Comment (warquys) [reply Synapse3.SynapseModule/Patching/Patches/EventMapPatches.cs]: I remove the event

**Pull request merged**